### PR TITLE
Replace index.html with Storybook

### LINKED ISSUE #4: Replace index.html with Storybook

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Comment by qur786** [comment]
CPT2204272349-1054x608

**Comment by maneetgoyal** [comment]
good speed ⚡

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  stories: ['../src/**/*.stories.(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+  ],
+  framework: '@storybook/react',
```
Comment (maneetgoyal) [inline .storybook/main.js:8]: why this line?

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
```
Comment (maneetgoyal) [inline .storybook/preview.js]: why this

File: package.json [hunk 3]
```diff
@@ -37,6 +39,14 @@
     "@babel/preset-react": "^7.16.7",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-node-resolve": "^13.2.1",
+    "@storybook/addon-actions": "^6.4.22",
+    "@storybook/addon-essentials": "^6.4.22",
```
Comment (maneetgoyal) [inline package.json]: use 6.5.x beta versions

File: .storybook/main.js (hunk 1 above)
Comment (qur786) [reply .storybook/main.js:8]: @maneetgoyal sir this line denotes the type of framework that we are using, It helps `Storybook` to load and build processes specific to framework. 
Source: https://storybook.js.org/docs/react/configure/overview

File: .storybook/preview.js (hunk 2 above)
Comment (qur786) [reply .storybook/preview.js]: not needed.

**Comment by qur786** [comment]
Getting these two errors in console.

Screenshot from 2022-04-28 09-45-26

**Comment by qur786** [comment]
> Getting these two errors in console.
> 
> Screenshot from 2022-04-28 09-45-26

For the pseudo class related  error, this https://stackoverflow.com/questions/57239807/the-pseudo-class-first-child-is-potentially-unsafe-when-doing-server-side-ren might be the answer.

File: .storybook/main.js (hunk 1 above)
Comment (maneetgoyal) [reply .storybook/main.js:8]: See if this should be added to d3 portfolio and app repo also, I think we aren't using it there but still things are working, try to research on this property helps and add it to the repos also if needed

Comment (maneetgoyal) [reply .storybook/main.js:8]: creating https://github.com/manufac-analytics/reactjs-calendar-heatmap/issues/8

**Comment by maneetgoyal** [comment]
new issues creates

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**